core/src/main/java/org/apache/druid/metadata/MetadataStorageActionHandler.java

```diff
@@ -32,13 +32,14 @@
 {
   /**
    * Creates a new entry.
-   * 
-   * @param id entry id
-   * @param timestamp timestamp this entry was created
+   *
```

Comment:
Why these changes, please don't modify code which is not related to your PR. The same for below.